plugin.video.nlhardwareinfo/resources/settings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
This file is not required if the add-on is not configurable. In fact it is better to ommit it than to provide an empty one